languages/swift/exercises/concept/loops/Sources/loops/loops.swift

```diff
@@ -0,0 +1,17 @@
+func timeToPrepare(drinks: [String]) -> Double {
+  fatalError("Please implement the timeToPrepare(drinks:) function")
+}
+
+func makeWedges(needed: Int, limes: [String]) -> Int {
+  fatalError("Please implement the makeWedges(needed:limes:) function")
+}
+
+func finishShift(minutesLeft: Int, remainingOrders: [[String]]) -> [[String]] {
+  fatalError("Please implement the finishShift(minutesLeft:remainingOrders:) function")
+}
+
+func orderTracker(orders: [(drink: String, time: String)]) -> (
+  beer: (first: String, last: String, total: Int)?, soda: (first: String, last: String, total: Int)?
```

Comment:
Is the `?` for an optional value? If so, should the `optionals` concept also be a prerequisite?

Reply:
Initially `strings` (which was converted into `string-components`) was a prerequisite and that had `optionals` as a prerequisite so I left it out. But the new exercise doesn't use `string-components` anymore, so I added `optionals` back in.